examples/ota-provider-app/esp32/main/BdxOtaSender.cpp

```diff
@@ -28,6 +28,32 @@ using chip::bdx::StatusCode;
 using chip::bdx::TransferControlFlags;
 using chip::bdx::TransferSession;
 
+CHIP_ERROR BdxOtaSender::InitializeTransfer(chip::FabricIndex fabricIndex, chip::NodeId nodeId)
```

Comment:
Cleaner to have one BDXOtaSender class, instead of two very similar ones. Ticket for the backlog, perhaps and maybe someone from Espressif can shed more light here?

Reply:
Ideally, we had refactored the OTA Provider in a similar fashion as OTA Requestor which could allow platform specific impl to be done with most code being common. This couldn't be achieved in the v1 timeframe so for the time being, the ota-provider-app for each platform has to make a pretty similar copy.